Tools/TestWebKitAPI/cocoa/TestWKWebView.h

```diff
@@ -85,6 +86,7 @@
 @interface TestWKWebView : WKWebView
 - (instancetype)initWithFrame:(CGRect)frame configuration:(WKWebViewConfiguration *)configuration processPoolConfiguration:(_WKProcessPoolConfiguration *)processPoolConfiguration;
 - (instancetype)initWithFrame:(CGRect)frame configuration:(WKWebViewConfiguration *)configuration addToWindow:(BOOL)addToWindow;
+- (instancetype)initWithFrame:(CGRect)frame configuration:(WKWebViewConfiguration *)configuration uiDelegate:(id<WKUIDelegate>)delegate addToWindow:(BOOL)addToWindow;
```

Comment:
For what it's worth, I'm not a fan of the fix you went with.

You could have fixed this by:
1. Using a regular WKWebView
2. Using a testWKWebView, call addToWindow after you've added your UIDelegate and make sure that the addToWindow implementation constructs the hostWindow if necessary.

Instead we have now yet another TestWKWebView init function, which is unlikely to be reused since callers can set the WKUIDelegate themselves.

Reply:
I'll switch back to the original state and adopt this approach.